homeassistant/__init__.py

```diff
@@ -45,6 +46,8 @@
 class HomeAssistant(object):
     """ Core class to route all communication to right components. """
 
+    arguments = None
```

Comment:
It is set on line 212 of __main__. It is meant to propagate cmd line arguments into HA so they can be recalled later. Like in the updater component.